compose/container.py

```diff
@@ -131,10 +133,18 @@ def environment(self):
         return dict(var.split("=", 1) for var in self.get('Config.Env') or [])
 
     @property
+    def status(self):
+        return self.get('State.Status')
```

Comment:
OK - that makes sense, but as we're not prioritising that use case right now, I'd rather not add it.